public/app.dev.json

```diff
@@ -20,5 +20,6 @@
         "company": "ScottLogic",
         "description": "Hosts BitFlux as an OpenFin application.",
         "name": "StockFlux"
-    }
+    },
+    "splashScreenImage": "splashscreen.png"
```

Comment:
Do relative paths work, or should we use absolute URIs? (I don't have anything against relative paths, just noticed the Angular version uses absolute paths.)